tephra-core/src/test/java/co/cask/tephra/TransactionContextTest.java

```diff
@@ -54,7 +53,7 @@
   @BeforeClass
   public static void setup() throws IOException {
     final Configuration conf = new Configuration();
-    conf.set(TxConstants.Persist.CFG_TX_SNAPHOT_CODEC_CLASSES, DefaultSnapshotCodec.class.getName());
+    conf.set(TxConstants.Persist.CFG_TX_SNAPHOT_CODEC_CLASSES, SnapshotCodecV2.class.getName());
```

Comment:
SnapshotCodecV4 is the latest.  We should use that unless we're explicitly testing upgrade of the formats.